.circleci/config.yml

```diff
@@ -3,11 +3,17 @@ version: 2
 jobs:
   build:
     docker:
-      - image: headspace/hs-node:8.9 # primary container
+      - image: node:10-alpine # primary container
         environment:
           NODE_ENV: test
     working_directory: ~/zuora-request-service
+    aliases:
+        - &add_authority
```

Comment:
what is this...?

Reply:
this is a solution for the error i was getting here: https://circleci.com/gh/HeadspaceMeditation/zuora-request-service/62?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification

after talking to ops and pulling from an example here:
https://github.com/HeadspaceMeditation/nodejs-platform/blob/master/.circleci/config.yml#L50